Http/Controllers/FreeScoutGPTController.php

```
@@ -417,6 +459,145 @@ public function generate(Request $request)
+                $systemPrompt = (!empty($ajax_cmd) ? $ajax_cmd : $settings->start_message);
+
+                $messages = [
+                    [
+                        'role' => 'system',
+                        'content' => $systemPrompt
+                    ]
+                ];
+
+                if ($settings->client_data_enabled) {
+                    $customerName = $request->get("customer_name");
+                    $customerEmail = $request->get("customer_email");
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$customerEmail'. (undefined)

(UnusedLocalVariable)